src/Components/bookHub/BookHub.js

```diff
@@ -13,32 +14,121 @@ const base = new Airtable({ apiKey: airtableConfig.apiKey })
   .base(airtableConfig.baseKey);
 
 function CardsDisplay() {
-  const [cards, setCards] = useState([]);
+  const [allBooks, setAllBooks] = useState([]);
+  const [filteredBooks, setFilteredBooks] = useState([]);
+  const [searchTerms, setSearchTerms] = useState('');
+  const [defaultSearch, setDefaultSearch] = useState(true);
 
   const getCards = () => {
     base('Book').select({ view: 'Grid view' }).all()
       .then((records) => {
-        setCards(records);
+        setAllBooks(records);
       });
   };
 
-  useEffect(getCards, []);
+  const isMatchTitleDescIdentity = (book) => {
+    // Filter function for books stored in 'books' state
+    /* FOR FUTURE DEV
+        - we may want to match by a combo of title, description, identity
+            - refactor this to |= the _.includes() calls for each field
+    */
+    if (!book) return false;
+    let title = book.get('title');
+    let desc = book.get('description');
+    let identity = book.get('identity');
+
+    // need to error check before using toLowerCase()
+    title = (title) ? title.toLowerCase() : '';
+    desc = (desc) ? desc.toLowerCase() : '';
+    identity = (identity) ? identity.toLowerCase() : '';
+
+    let match = false;
+    match = title.includes(searchTerms)
+    || desc.includes(searchTerms)
+    || identity.includes(searchTerms);
+    return match;
+  };
+
+  const Filter = (table, field) => new Promise((resolve, reject) => {
```

Comment:
Here too, can you rename the function to be more connotative? This is filtering by creator right